src/Microsoft.DocAsCode.MarkdigEngine/Validation/ValidationExtension.cs

```diff
@@ -25,6 +28,13 @@ public void Setup(MarkdownPipelineBuilder pipeline)
 
             pipeline.DocumentProcessed += document =>
             {
+                if (InclusionContext.IsInclude 
+                && (string.Equals(Path.GetExtension(InclusionContext.RootFile?.ToString()), ".yml", StringComparison.OrdinalIgnoreCase) 
+                || string.Equals(Path.GetExtension(InclusionContext.RootFile?.ToString()), ".yaml", StringComparison.OrdinalIgnoreCase)))
+                {
+                    document.SetData("yamlMime", YamlMime.ReadMime(InclusionContext.RootFile?.ToString()));
```

Comment:
yamlMime may not be a good name as it is actually the MIME of the **root file**

shall we consider `rootFileYamlMime`?

Reply:
Sure, rename it.